packages/stark-ui/assets/styles/_base.scss

```diff
@@ -99,3 +99,190 @@ body {
 .flex-fill {
   flex: 1;
 }
+
+/* stark-layout */
+$stark-layout-spacing: 15px;
+.stark-layout {
+  display: flex;
+  flex-wrap: wrap;
+  overflow: hidden;
+
+  margin: -$stark-layout-spacing;
+
+  > * {
+    flex: 1;
+    margin: $stark-layout-spacing;
+    box-sizing: border-box;
+
+    mat-form-field {
+      width: 100%;
+    }
+
+    @media #{$mobile-only-query} {
+      min-width: calc(100% - (#{$stark-layout-spacing} * 2));
+    }
+  }
+}
+
+/********** BUTTONS **********/
+/*@mixin stark-base-color($color) {*/
+button:not([disabled]),
+.mat-button:not([disabled]) {
+  &.stark-outline,
+  &.stark-delete {
+    min-height: 32px;
+    min-width: 86px;
+    line-height: 32px;
+    background-color: #fff;
+    padding: 0 4px;
+
+    &:active,
+    &:hover {
+      background-color: $offwhite;
+    }
+  }
+
+  &.stark-outline {
+    padding: -1px 4px;
+    border-style: solid;
+    border-width: 2px;
+
+    &.mat-primary {
+      border-color: $md-primary;
+      color: $md-primary;
+
+      & svg {
+        fill: $md-primary;
```

Comment:
could the colors be moved to `packages/stark-ui/assets/theming/_base-theme.scss` and use 
`mat-color($primary-palette, 500)` etc. 😄